src/site/content/en/blog/sanitizer/index.md

```diff
@@ -0,0 +1,380 @@
+
+{% Aside 'caution' %}
+Details of vulnerabilities can be found below.
+
+[Mutation XSS via namespace confusion - DOMPurify < 2.0.17 bypass - research.securitum.com](https://research.securitum.com/mutation-xss-via-mathml-mutation-dompurify-2-0-17-bypass/)
+{% endAside %}
+
+HTML also needs **context** to be parsed. For example, `<td>` makes sense in `<table>`, but not in `<div>`. Since `DOMPurify.sanitize()` only takes a string as an argument, the parsing context had to be guessed.
+
+(A detailed explanation is in the [Explainer](https://github.com/otherdaniel/purification/blob/strings-explainer/explainer-strings.md))
+
+The [Sanitizer API](https://wicg.github.io/sanitizer-api/) has learned from the DOMPurify experience and is designed to eliminate the need for double parsing and to clarify the parsing context as follows.
+
+
+## Current status
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Step</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>1. Create explainer</td>
+        <td><a href="https://github.com/WICG/sanitizer-api/blob/main/README.md">Complete</a></td>
+      </tr>
+      <tr>
+        <td>2. Create draft of specification</td>
+        <td><a href="https://wicg.github.io/sanitizer-api/">Complete</a></td>
+      </tr>
+      <tr>
+        <td>3. Gather feedback and iterate on design</td>
+        <td><a href="https://docs.google.com/document/d/1kBpk-dhCukPphfH_8vkOgdUDcM9bqVuhsK0pRZzwapM/edit?resourcekey=0-4tFsd4s6aLbvvr2ys_9F4A#Feedback">In progress</a></td>
+      </tr>
+      <tr>
+        <td>4. Origin trial</td>
+        <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/OrWQnXVQJ0A/m/TbF-0Dw3BQAJ">Ready for Trial</a></td>
+      </tr>
+      <tr>
+        <td>5. Launch</td>
+        <td>Not started</td>
+      </tr>
+    </tbody>
+    <caption>Current Status</caption>
+  </table>
+</div>
+
+
+## How to enable the Sanitizer API
+
+
+### Enabling via about://flags or CLI option
+
+
+#### Chrome
+
+Chrome is in the process of implementing the Sanitizer API, and when Chrome 93 is released, you can try out the behavior by enabling `about://flags/#enable-experimental-web-platform-features`. In earlier versions of the Canary/Dev channel, you can enable it via `--enable-blink-features=SanitizerAPI` and try it out for yourself right now. More details about using a flag can be found [here](https://www.chromium.org/developers/how-tos/run-chromium-with-flags).
+
+
+#### Firefox
+
+Firefox also implements the Sanitizer API. You can set the `dom.security.sanitizer.enabled` flag to `true` in `about:config` and it works as experimental.
+
+### Feature detection
+
```

Comment:
I think the note about using DOMPurify as a fallback should go here

Reply:
DOMPurify should be mentioned in stand alone section in this article.